editors/code/package.json

```diff
@@ -354,35 +354,122 @@
         ],
         "configuration": [
             {
-                "title": "general",
+                "title": "Rust Analyzer"
+            },
+            {
+                "title": "Assist"
+            },
+            {
+                "title": "Cache Priming"
+            },
+            {
+                "title": "Cargo"
```

Comment:
This part forces the categories to be ordered lexicographically. Otherwise they are ordered based the order which they appear in the file. It's possible that this was intentional, in which case I'd remove these lines.